pkg/cmd/importcmd/import.go

```diff
@@ -707,68 +712,66 @@ func (options *ImportOptions) CreateNewRemoteRepository() error {
 	repoURL := repo.HTMLURL
 	options.GetReporter().PushedGitRepository(repoURL)
 
+	return options.AddBotAsCollaborator()
+}
+
+// AddBotAsCollaborator adds the pipeline bot as collaborator to the repository
+func (options *ImportOptions) AddBotAsCollaborator() error {
 	githubAppMode, err := options.IsGitHubAppMode()
 	if err != nil {
 		return err
 	}
 
 	if !githubAppMode {
-
 		// If the user creating the repo is not the pipeline user, add the pipeline user as a contributor to the repo
 		if options.PipelineUserName != options.GitUserAuth.Username && options.GitServer != nil && options.GitServer.URL == options.PipelineServer {
 			// Make the invitation
-			err := options.GitProvider.AddCollaborator(options.PipelineUserName, details.Organisation, details.RepoName)
+			err := options.GitProvider.AddCollaborator(options.PipelineUserName, options.Organisation, options.AppName)
+			if err != nil {
+				return err
+			}
+
+			authConfigSvc, err := options.GitAuthConfigService()
```

Comment:
What tricked me here: Previously `options.GitLocalAuthConfigService()` was used (for quickstart) which used the local git credentials. In order to retrieve the pipeline bot's credentials from vault `options.GitAuthConfigService()` must be used and the credentials obtained using `authConfigSvc.Config().GetPipelineAuth()`.